app/terminal/terminal.css

```
@@ -127,9 +127,7 @@
   display: flex;
   align-items: center;
   padding: 0 8px;
-}
-
-.terminal .no-results {
+  color: white;
   opacity: 0.3;
 }
```

- Keep `opacity: 0.3` within `.no-results` (i.e. only when "No results" is displayed), otherwise the text is too light when displaying "1 of 3" etc.
- For light terminal theme, text should be black

```suggestion
  color: white;
}

.terminal.light-terminal .search-result-count {
  color: black;
}

.terminal .no-results {
  opacity: 0.3;
}
```